proxy/src/main/java/net/md_5/bungee/netty/ChannelWrapper.java

```diff
@@ -38,30 +35,44 @@ public void setVersion(int protocol)
 
     public void write(Object packet)
     {
-        if ( !closed )
+        if ( !isClosed() )
         {
             if ( packet instanceof PacketWrapper )
             {
                 ( (PacketWrapper) packet ).setReleased( true );
-                ch.write( ( (PacketWrapper) packet ).buf, ch.voidPromise() );
+                ch.write( ( (PacketWrapper) packet ).buf ).addListener( ChannelFutureListener.FIRE_EXCEPTION_ON_FAILURE );
             } else
             {
-                ch.write( packet, ch.voidPromise() );
+                ch.write( packet ).addListener( ChannelFutureListener.FIRE_EXCEPTION_ON_FAILURE );
             }
```

Comment:
Again, you should really use `voidPromise()` as it as the same effect without torturing the garbage collector,